scripts/single.php

```diff
@@ -43,4 +52,3 @@
         echo 'Message: ' .$e->getMessage();
     }
     $web_driver->quit();
-?>
```

Comment:
Need to have php closing braces

Reply:
ack, added now.. somehow tests ran even with this missing.